src/features/book/Bookslice.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/neTvlShlUagV28v2Fadm';
+
+const initialState = {
+  books: [],
+};
+
+export const bookPost = createAsyncThunk(
+  'post/bookPost',
+  async (bookData, thunkAPI) => {
+    try {
+      const response = await axios.post(`${URL}/books`, bookData);
+      return response.data;
+    } catch (error) {
+      return thunkAPI.rejectWithValue(
+        error?.data.message || 'Something went wrong. Please check later!',
+      );
+    }
+  },
+);
+
+export const fetchBooks = createAsyncThunk('books/fetchBooks', async (_, thunkAPI) => {
+  try {
+    const response = await axios(`${URL}/books`);
+    return response.data;
+  } catch (error) {
+    return thunkAPI.rejectWithValue(
+      error?.data.message || 'Something went wrong. Please check later!',
+    );
+  }
+});
+
+export const removeBookFromAPI = createAsyncThunk(
+  'books/deleteBook',
+  async (id, thunkAPI) => {
+    try {
+      const response = await axios.delete(`${URL}/books/${id}`);
+      return response.data;
+    } catch (error) {
+      return thunkAPI.rejectWithValue(
+        error?.data.message || 'Something went wrong. Please check later!',
+      );
+    }
+  },
+);
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      // const bookData = actions.payload;
+      state.books.push(action.payload);
```

Comment:
- [ ] Let's try not to have commented unused code in our codebase as they make our code dirty and not professional. Kindly remove all unused codes from your codebase 👍🏽

Reply:
resolved! ✔️